integreat_cms/cms/templates/content_revisions.html

```diff
@@ -108,10 +106,10 @@ <h1>{{ page_translation.title }}</h1>
             </div>
         </div>
         <div class="w-full p-4 flex justify-end gap-4 action-buttons">
-            {% if not page.archived %}
-                {% has_perm 'cms.change_page_object' request.user page as can_edit_page %}
-                {% has_perm 'cms.publish_page_object' request.user page as can_publish_page %}
-                {% if can_publish_page %}
+            {% if not content.archived %}
+                {% has_perm 'cms.change_{{ content_name }}_object' request.user content as can_edit %}
+                {% has_perm 'cms.publish_{{ content_name }}_object' request.user content as can_publish %}
```

Comment:
The `cms.change_{{ content_name }}_object` permission only exists for pages, so we need to use a different syntax for all other content models.